pkg/cli/clusteroperator/restartkubelet/restart-pod-template.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: restart-kubelet-pod-
+spec:
+  restartPolicy: Never
+  containers:
+    - command:
+        - sleep
+        - "10"
+      image: registry.redhat.io/openshift4/ose-must-gather:latest
```

Comment:
Why this image?

Reply:
> Why this image?

Same default as our debugging today.

Reply:
isn't that `"openshift/tools:latest"` as in https://github.com/openshift/oc/blob/641d6f024a7a8f1a66ed311f3b8070c8a4157570/pkg/cli/debug/debug.go#L388 ?